splice_access_api/src/main/java/com/splicemachine/access/configuration/SQLConfiguration.java

```diff
@@ -235,7 +235,10 @@
     public static final int DEFAULT_RECURSIVE_QUERY_ITERATION_LIMIT = 20;
 
     public static final String METADATA_RESTRICTION_ENABLED = "splice.metadataRestrictionEnabled";
-    public static final boolean DEFAULT_METADATA_RESTRICTION_ENABLED = true;
+    public static final String METADATA_RESTRICTION_DISABLE = "DISABLE";
```

Comment:
Could you change the name from METADATA_RESTRICTION_DISABLE to METADATA_RESTRICTION_DISABLED, and the value from "DISABLE" to "DISABLED"?

Reply:
I'm also curious why not to follow the pattern of `METADATA_RESTRICTION_ENABLED` to use `splice.metadataRestrictionDisabled`? Wait...why can't we use `METADATA_RESTRICTION_ENABLED`?

Reply:
@yxia92 Fixed this.

@jaceklaskowski  `METADATA_RESTRICTION_ENABLED` is a configuration name and `METADATA_RESTRICTION_DISABLED` is the value of this configuration.